test/ossl_shim/async_bio.cc

```diff
@@ -118,7 +118,7 @@ static int AsyncFree(BIO *bio) {
   return 1;
 }
 
-static long AsyncCallbackCtrl(BIO *bio, int cmd, bio_info_cb fp) {
+static long AsyncCallbackCtrl(BIO *bio, int cmd, BIO_info_cb fp) {
```

Comment:
as long as you're here please put the function-open curly on its own line.

Reply:
sure, no problem.